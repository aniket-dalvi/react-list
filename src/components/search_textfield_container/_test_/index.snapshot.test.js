import React from "react";
import render from "react-test-renderer";
import SearchTextfieldContainer from "../../search_textfield_container";

describe("snapshot test for search textfield container", () => {
    test("snapshot: ", () => {
        const app = render.create(<SearchTextfieldContainer/>).toJSON();
        expect(app).toMatchSnapshot();
    });
})