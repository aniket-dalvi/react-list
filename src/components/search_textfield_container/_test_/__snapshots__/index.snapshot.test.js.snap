// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for search textfield container snapshot:  1`] = `
<div
  className="textfield-wrapper"
>
  <label
    className="header"
  >
    Friends List
  </label>
  <input
    onChange={[Function]}
    onKeyPress={[Function]}
    placeholder="Enter your friend's name"
    value=""
  />
</div>
`;
